python/pyspark/rdd.py

```diff
@@ -123,6 +129,13 @@ def _load_from_socket(port, serializer):
         sock.close()
 
 
+def ignore_unicode_prefix(f):
```

Comment:
Please add docstring for this function. It is not clear what it does based on the method name.